ast/location/location.go

```diff
@@ -87,3 +93,34 @@ func (loc *Location) Compare(other *Location) int {
 	}
 	return 0
 }
+
+func (loc *Location) MarshalJSON() ([]byte, error) {
+	// structs are used here to preserve the field ordering of the original Location struct
+	var data interface{}
```

Comment:
[nit] We can use `any` now, I think. Introduced in 1.18.

Reply:
👍 I've accepted your edit below instead in 4e213719e6e0241a4a0e6d1572c2dd84acb115c6
🙂